test/line-clamp.js

```diff
@@ -0,0 +1,33 @@
+import { setup } from "./_helpers.js";
+import { describe, expect, test } from "vitest";
+import { getNumericArrayInRange } from '#utils';
+
+
+setup();
+
+describe('line-clamp', () => {
+  test('should render styles for static classes', async ({ uno }) => {
+    const staticClasses = getNumericArrayInRange(1, 9).map(n => `line-clamp-${n}`);
+
+    const { css } = await uno.generate(staticClasses);
+    expect(css).toMatchInlineSnapshot(`
```

Comment:
Why inline snapshot here and removing all inline snapshot in `position.js`? Many of those inline snapshots there are shorter than this one. 😕

Reply:
OK! 

To be clear if I post a question I mean it as a genuine question, there might be a good for something that I don't understand - so please don't feel like some immediate action is needed!

Reply:
Right, got it.